commands/general/dev.js

```diff
@@ -17,12 +17,12 @@ exports.run = async (handler, message, args, pre) => {
 			.setAuthor(member.displayName, message.author.displayAvatarURL())
 			.setTitle('Developer Role Application')
 			.addField('User ID:', message.member.id)
-			.addField('GitHub URL:', args.github)
+			.addField('GitHub URL:', args['GitHub URL'])
```

Comment:
Is there any real need to rename this? Yes it's more ambiguous but it wasn't really that ambiguous in the first place. We could go for something like `githubURL` or `gitHubURL` as a compromise, I don't see the point of using an English-like object key.

Reply:
The idea was that it would show up more clearly in the help's usage information. But `githubURL` works just as well, I suppose.